test/object-expressions/property-name.js

```diff
@@ -16,10 +16,16 @@
 
 var expect = require('expect.js');
 var ShiftParser = require('../..');
+var assertParseFailure = require('../assertions').assertParseFailure;
+
 
 describe("Parser", function () {
   describe("property name", function () {
     expect(ShiftParser.default("({0x0:0})").body.statements[0].expression.properties[0].name.value).to.be("0");
     expect(ShiftParser.default("({2e308:0})").body.statements[0].expression.properties[0].name.value).to.be("" + 1 / 0);
+
+    assertParseFailure(function () { ShiftParser.default("({__proto__:42,__proto__:43})") });
+    assertParseFailure(function () { ShiftParser.default("use strict; ({__proto__:42,__proto__:43})") });
+    assertParseFailure(function () { ShiftParser.default("({__proto__:42},{__proto__:43})") });
```

Comment:
removed these and added some success tests for __proto__, setter, and getter here.